content/22.vaccines.md

```diff
@@ -247,6 +255,11 @@ Furthermore, the producers of Gam-COVID-Vac state that they can produce the vacc
 ### Protein Subunit Vaccines
 
 <!--Do we need to add this to the figure? Or does it fall in one of the other categories?-->
+Compared to the _inactivated whole virus vaccine_, these protein subunit vaccines isolate a single protein of the virus and use it to stimulate the immune system.
```

Comment:
Is there a reason to italicize this? Just checking!

Reply:
Its italicized because I was referring to the earlier section header. But perhaps this is not the best way to do that. I don't want to confuse the reader having it italicized.